internal/relocated/bundle.go

```diff
@@ -58,6 +69,17 @@ func (b *Bundle) writeRelocationMap(dest string, mode os.FileMode) error {
 	return ioutil.WriteFile(dest, d, mode)
 }
 
+// writeRepoDigest store the repo digest to a file as plain text.
+func (b *Bundle) writeRepoDigest(dest string, mode os.FileMode) error {
+	if b.RepoDigest == "" {
+		if _, err := os.Stat(dest); os.IsNotExist(err) {
+			return nil
+		}
+		return os.Remove(dest)
```

Comment:
Removing the digest in the `writeRepoDigest` function looks weird 🤔
Maybe we should put it in a sub function like the following, to make it more obvious we will clean it?
```go
func (b *Bundle) writeRepoDigest(dest string, mode os.FileMode) error {
    if b.RepoDigest == "" {
        return cleanRepoDigest(dest)
    }
    return ioutil.WriteFile(dest, []byte(b.RepoDigest), mode)
}

func cleanRepoDigest(dest string) error{
	if _, err := os.Stat(dest); os.IsNotExist(err) {
		return nil
	}
	return os.Remove(dest)
}
```